packages/eslint-config-angular/promise/index.js

```diff
@@ -22,6 +22,8 @@ module.exports = {
           'error',
           { ignoreVoid: true, ignoreIIFE: true },
         ],
+        // for compat with @typescript-eslint/no-floating-promises ignoreVoid
+        'no-void': ['error', { allowAsStatement: true }],
```

Comment:
Please, add unit test for that

examples:
https://github.com/Tinkoff/linters/tree/master/packages/eslint-config-angular/test

Reply:
done